iceoryx_examples/README.md

```diff
@@ -12,6 +12,7 @@
 |[waitset_in_c](./waitset_in_c/)             | Waiting for events like arrival of data using C                           | :star::star:       |
 |[iceensemble](./iceensemble/)               | Using multiple publishers for one topic                                   | :star::star:       |
 |[singleprocess](./singleprocess/)           | Communicating in a single process between threads                         | :star::star:       |
+|[user_header](./user_header/)               | Using a custom user-header for additional meta-information                | :star::star:       |
```

Comment:
Maybe it should be added that this meta-infos are attached to the sample containing the user payload